apps/client/src/locales/messages.pl.xlf

```diff
@@ -6069,7 +6069,7 @@
       </trans-unit>
       <trans-unit id="1677491810118195784" datatype="html">
         <source>self-hosting</source>
-        <target state="new">self-hosting</target>
+        <target state="translated">wlasny-hosting</target>
```

Comment:
Your suggestion `wlasny_hosting` was actually correct for `snake-case`!
I will change the hint to `kebab-case` in another PR 🤓